docs/api_docs/DecisionTree.md

```diff
@@ -0,0 +1,26 @@
+# DecisionTree Class
+
+The decision tree lies at the core of adaXT. It uses the following
+four individual components to construct specific types of decision
+trees that can then be applied to data.
+
+- [Criteria](../criteria/criteria.md)
+- [LeafBuilder]()
+- [Prediction]()
+
+Instead of the user specifying all three components individually, it
+is also possible to only specify the ``tree_type``, which then
+internally selects the corresponding default components for several
+established tree-algorithms.
+
+For more advanced modifications, it might be necessary to change the
+splitting itself. This can be done by passing a custom [Splitter
+class](../splitter/splitter.md).
```

Comment:
```suggestion
class](Splitter.md).
```
You want a link to the given Splitter markdown, which is in the same folder as the 
current markdown file.